### PR TITLE
Storageareas navigation

### LINKED ISSUE #21: (storageareas Branch) gapless events not working
Gapless events from the replay selector on events page dose not work. I'm on branch storageareas and the latest as of 26/09/2017 at 23:12 my time and I'm using mp4 video for all events no images.

### PR DESCRIPTION
**Pull request by digital-gnome**
This fixes most missing navigation in storageareas.  All of the replay modes work.  There is a previous and next button borrowed from DVR controls.  I did not include speed controls because they are built in to video.js.  My styling probably leaves something to be desired.  This fixes #21 

Unfortunately is still uses mootools because I just extended the existing next and previous functionality.  Switching that to jquery was not as trivial as I thought.

### CODE REVIEW

File: web/skins/classic/css/classic/views/event.css [hunk 1]
```diff
@@ -106,32 +106,32 @@
 #monitorStatus #monitorState {
 }
 
-#dvrControls {
+dvrControls {
```
Comment (connortechnology) [inline web/skins/classic/css/classic/views/event.css]: This is not correct... the # tells it to select by id.. without the # it is not valid css.

File: web/skins/classic/views/event.php [hunk 2]
```diff
@@ -167,7 +167,17 @@
         var duration = <?php echo $Event->Length() ?>, startTime = '<?php echo $Event->StartTime() ?>';
 
         addVideoTimingTrack(document.getElementById('videoobj'), LabelFormat, monitorName, duration, startTime);
+
+        nearEventsQuery( eventData.Id );
+        vjsReplay(<?php echo (strtotime($Event->StartTime()) + $Event->LEngth())*1000 ?>);
         </script>
```
Comment (connortechnology) [inline web/skins/classic/views/event.php]: LEngth typo

Comment (digital-gnome) [reply web/skins/classic/views/event.php]: Fixed

File: web/skins/classic/css/classic/views/event.css (hunk 1 above)
Comment (digital-gnome) [reply web/skins/classic/css/classic/views/event.css]: I got it in my head that class selector was a bare name.  Meant to . those.  Checking if everything works as a class.  This explains why I thought your branch was styling the buttons differently than main.  I'd already broken it before I checked.

**Comment by digital-gnome** [comment]
Fixed the typo and fixed the css.  Added jquery bit to also set class.

**Pull request merged**